contrib/go-redis/redis.v7/redis.go

```diff
@@ -72,6 +72,7 @@ func additionalTagOptions(client redis.UniversalClient) []ddtrace.StartSpanOptio
 		if opt.Addr == "FailoverClient" {
 			additionalTags = []ddtrace.StartSpanOption{
 				tracer.Tag("out.db", strconv.Itoa(opt.DB)),
+				tracer.Tag(ext.RedisDatabaseIndex, opt.DB),
```

Comment:
Do we still need to have both? Is there a migration plan to remove these deprecated tags?

Reply:
For now we do need to have both, we are still discussing the best way to proceed deprecating old tags without affecting users

Reply:
Can the existing automatic tag renaming system in the backend not handle this tag? (admittedly I'm not super familiar with it, so it's possible there's a totally normal reason that it can't be used here)